torchx/c_src/torchx.cpp

```diff
@@ -1,5 +1,5 @@
 #include <torch/torch.h>
-#include <ATen/BatchedTensorImpl.h>
+#include <ATen/LegacyBatchedTensorImpl.h>
```

Comment:
Because the only mandatory change in this file is this, we could in theory still support libtorch 1.12 and therefore earlier CUDA as well. Thoughts?

Reply:
Supporting both for now sounds good to me.

Reply:
Yeah, especially now that I found out that macOS is still only supported for x86 by default.

I'll ping the maintainer for that repo we fetch from to see if we can get 2.0 there too